src/Components/Artwork/GridItem.tsx

```diff
@@ -34,6 +51,7 @@ interface Props extends React.HTMLProps<ArtworkGridItemContainer> {
 
 interface State {
   isMounted: boolean
+  isImageLoaded: boolean
```

Comment:
Should the `<ArtworkGrid>` take a `lazyLoad` configuration prop? I'm thinking of pages like: https://www.artsy.net/collect where the the grid is at the top of the screen and you'd want to be able to see the images on load and with JS disabled.

Reply:
I plan to have a prop for `ArtworkGrid` that accepts a number of images to eagerly load. So for the collect page 9 images on desktop, 6 images on mobile would be ideal. For artist/artwork page 3 to 6 images makes sense to eagerly load on desktop, but you'd probably want none on mobile. 

As per the js disabled bit... that should generally be handled by the lib. I'll look into if it is, if not, I'll PR it.

Reply:
Is there no way for the lib to just eager load above the fold? (Not sure if the above thread is about that.)

Reply:
Nope. I mean, when you load the page and the js initializes it will auto load anything in view (and within the threshold). But the above the fold loading decision has to be made at SSR which falls back to the traditional not knowing the user’s browser size.

Reply:
I've manually configured the lazy-loading values for each case. For artwork grids completely above the fold (think collect, works-for-you) there are 9 preloaded images. That seems to cover a pretty tall screen.